### PR TITLE
Step 5 - Include Datastore

### PR DESCRIPTION
**Pull request by lornaarmstrong**
Note: I have commented some TODOs for aspects that aren't finished, but the main functionality is there.

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -18,15 +18,21 @@ function loadPage() {
   getComments();
 }
 
+//TO DO Change the += format to use Node.appendChild()
```
Comment (MostafaGabriel) [inline portfolio/src/main/webapp/script.js:21]: nit: Please leave a blank space after the `//` and use a single word for TODO :)
It's also useful to assign the TODO to someone specially in a shared project so it'd be clear who should do it, e.g. `TODO(username): add more details`

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -19,34 +19,69 @@
...
 
 /** Servlet that returns some comments.*/
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-  ArrayList<ArrayList<String>> messages = new ArrayList<>();
-
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
```
Comment (MostafaGabriel) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:42]: given that it's also used in `doPost`, please consider having it as a class variable and initialize it once inside the `init()` method

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -18,15 +18,21 @@ function loadPage() {
...
-        commentHTML += "<p id = \"commentText\">" + comments[i][2] + "</p>";
+    if (messages.length == 0){
+       commentHTML += "<p>There are no comments yet.</p>"
+    } else {
+      for (var i = 0; i < messages.length; i++){
+        commentHTML += "<div id = \"user-comment\">";
+        commentHTML += "<h4 id = \"commentor-name\">" + messages[i]["firstName"] + " " + messages[i]["lastName"] + "</h4>";
+        commentHTML += "<h5 id = \"comment-date\">" + messages[i]["dateTime"] + "</h5>";
+        commentHTML += "<p id = \"comment-text\">" + messages[i]["message"] + "</p>";
         commentHTML += "</div>"
```
Comment (MostafaGabriel) [inline portfolio/src/main/webapp/script.js:34]: bonus: consider using `concat()` method here. [Ref](https://www.w3schools.com/jsref/jsref_concat_string.asp)

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (lornaarmstrong) [reply portfolio/src/main/webapp/script.js:21]: Will do, thanks!

File: portfolio/src/main/webapp/script.js (hunk 3 above)
Comment (lornaarmstrong) [reply portfolio/src/main/webapp/script.js:34]: Thanks - I'll read into this!

**Pull request merged**